tools/packaging/packaging.psm1

```diff
@@ -242,8 +242,8 @@ function Start-PSPackage {
                     ProductVersion = $Version
                     AssetsPath = "$PSScriptRoot\..\..\assets"
                     LicenseFilePath = "$PSScriptRoot\..\..\assets\license.rtf"
-                    # Product Guid needs to be unique for every PowerShell version to allow SxS install
-                    ProductGuid = New-Guid
+                    # Product Code needs to be unique for every PowerShell version to allow SxS install
```

Comment:
This is not correct either...  but here just remove the `to allow SxS install` part.